src/adapter/src/catalog.rs

```diff
@@ -2075,7 +2077,8 @@ impl Source {
     /// Returns whether this source ingests data from an external source.
     pub fn is_external(&self) -> bool {
         match self.data_source {
-            DataSourceDesc::Ingestion(_) => true,
+            DataSourceDesc::Ingestion(_)
+            | DataSourceDesc::Webhook => true,
```

Comment:
You know what the consequence of this state is? There's a chance this isn't "true" because we are not e.g. using a `CONNECTION`.

Reply:
It doesn't appear to actually be called by anyone, and deleting the method returns no errors when I `cargo c --tests` the entire workspace 🤔